pkgs/top-level/all-packages.nix

```diff
@@ -11638,7 +11638,8 @@ with pkgs;
   oracleXE = callPackage ../servers/sql/oracle-xe { };
 
   softether_4_18 = callPackage ../servers/softether/4.18.nix { };
```

Comment:
So we need to keep 4.18 around?

Reply:
I add `services.softether.package` option for this, we could use this version when we got something wrong.